src/eval.jl

```diff
@@ -20,6 +20,17 @@ function getmodule(data, pos)
   getthing("$main.$sub", getthing(main, Main))
 end
 
+function ends_with_semicolon(line)
+    match = rsearch(line, ';')
+    if match != 0
+        for c in line[(match+1):end]
+            isspace(c) || return c == '#'
+        end
+        return true
+    end
+    return false
+end
```

Comment:
Is this the same function that exists in Base? If so, it'd be better to just call that.